BackofficeBundle/Resources/public/coffee/tab/TabView.coffee

```diff
@@ -19,11 +19,11 @@ TabView = OrchestraView.extend(
     $('.js-widget-title', @options.domContainer).html @options.title
 
 
-  addPanel: (title, id, view, active, position) ->
+  addPanel: (title, id, view, active, badge, position) ->
     if @$tabNav? and @$tabContent?
-      @createPanel(title, id, view, active, position)
+      @createPanel(title, id, view, active, badge, position)
     else
-      @options.panels.push {title: title, id: id, view: view, active: active, position: position}
+      @options.panels.push {title: title, id: id, view: view, active: active, badge, position: position}
```

Comment:
You can't use the value badge without a key in associative context.
This key should be use line 57.
As badge is not mandatory, this option should be after position.
It would be easier to use as in https://github.com/open-orchestra/open-orchestra-media-admin-bundle/blob/master/MediaAdminBundle/Resources/public/coffee/SuperboxLoad.coffee Line 29.
@clement-michelet